### PR TITLE
deps(package): update package.json versions for vulnerabilities

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by CrispusDH** [comment]
You might need to update here as well? https://github.com/cnishina/protractor/blob/sourcemap/circle.yml#L57

**Comment by cnishina** [comment]
There's probably something in pretest / gulpfile that should be updated. I (for some reason) backed out that change.

File: circle.yml [hunk 1]
```diff
@@ -54,8 +54,8 @@ jobs:
           name: Selenium Start
           background: true
           command: |
-            ./node_modules/.bin/webdriver-manager update
-            ./node_modules/.bin/webdriver-manager start
+            ./node_modules/.bin/webdriver-manager update --versions.chrome=2.43
```
Comment (heathkit) [inline circle.yml]: Why do we need to pin the chromedriver version? Is Circle stuck on Chrome 69 for some reason?

Comment (cnishina) [reply circle.yml]: Yes CircleCI is giving me problems. This could be resolved if we could download and run a Chromium  version.  For now this is the fix.

**Comment by cnishina** [comment]
Closing this. There have been other PRs that were submitted that do audit checks.

**Pull request closed without merging**